fabtests/pytest/sm2/conftest.py

```diff
@@ -9,8 +9,3 @@
                                         pytest.param("host_to_neuron", marks=pytest.mark.neuron_memory)])
 def memory_type(request):
     return request.param
-
-# TODO Restore delivery_complete when completed
-@pytest.fixture(scope="module", params=["transmit_complete"])
-def completion_type(request):
-    return request.param
```

Comment:
Still need to separate testing changes

Reply:
done, this accidentally got messed up in a rebase of mine.